lib/presentation/widgets/sentiment_details.dart

```diff
@@ -18,38 +15,17 @@ class SentimentDetails extends StatelessWidget {
         if (state is LoadedSentimentState) {
           return _SentimentCard(
             children: [
-              SentimentChart(
-                data: <SentimentChartData>[
-                  SentimentChartData(
-                    'Positive sentiment',
-                    state.positiveSentiment,
-                    label: 'Positive',
-                    color: const Color.fromARGB(255, 38, 255, 89),
+              Flexible(
```

Comment:
Is there a reason you got rid of the Pie Chart and replaced it with the sentiment overview? In any case, the sentiment overview should be in a seperate widget called "SentimentOverview". Please revert this change